packages/autorest.python/autorest/m4reformatter/__init__.py

```diff
@@ -665,6 +671,9 @@ def _update_body_parameter_helper(
                 body_param["clientDefaultValue"] = body_type["value"]
         body_param["flattened"] = flattened
         body_param["isPartialBody"] = is_partial_body
+        body_param["restApiName"] = body_param["restApiName"] or to_lower_camel_case(
```

Comment:
it doesn't seem like we need this since body params don't actually have rest api names, they are just the body part of a request

Reply:
@iscai-msft In example files, they usually have a name like [here](https://github.com/Azure/azure-rest-api-specs/blob/88479e59185400417c4aa0fc369accfdf8c251ba/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-05-01/examples/GenerateCostDetailsReportByBillingAccountEnterpriseAgreementCustomerAndBillingPeriod.json#L5). We need it to extract parameter value :https://github.com/Azure/autorest.python/blob/d94843f4b088fb4394dd9ce984f8561e3ea721b5/packages/autorest.python/autorest/codegen/serializers/sample_serializer.py#L104-L105